clang/utils/analyzer/exploded-graph-rewriter.py

```diff
@@ -479,12 +479,15 @@ def add_raw_line(self, raw_line):
 # A visitor that dumps the ExplodedGraph into a DOT file with fancy HTML-based
 # syntax highlighing.
 class DotDumpVisitor:
-    def __init__(self, do_diffs, dark_mode, gray_mode, topo_mode, dump_dot_only):
+    def __init__(
+        self, do_diffs, dark_mode, gray_mode, topo_mode, dump_dot_only, dump_html_only
+    ):
```

Comment:
assert added to the constructor of `DotDumpVisitor`

Reply:
I used to consider merging these two options. But this would affect some test cases.
I am afraid of breaking some other places that I do not know, I hence added a new option in the first version.
If only the test cases will be affected, I can re-impl it in the merged way.